modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/TabStoppingTest.ts

```diff
@@ -0,0 +1,33 @@
+import { FocusTools, Keys } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { SugarDocument } from '@ephox/sugar';
+import { TinyContentActions, TinyHooks, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('webdriver.tinymce.themes.silver.editor.TabStoppingTest', () => {
```

Comment:
I believe this is the wrong description as this is not a webdriver test but rather a browser test. Right ?
 
```suggestion
describe('browser.tinymce.themes.silver.editor.TabStoppingTest', () => {
```

Reply:
Correct, missed changing that.